src/Spice86.Core/Emulator/Devices/Input/Keyboard/KeyboardCommand.cs

```diff
@@ -0,0 +1,17 @@
+namespace Spice86.Core.Emulator.Devices.Input.Keyboard;
+
+/// <summary>
+/// Keyboard commands that can be sent to the keyboard controller. <br/>
+/// https://www.win.tue.nl/~aeb/linux/kbd/scancodes-11.html#kcc60
+/// </summary>
+public enum KeyboardCommand : byte {
+    /// <summary>
+    /// No command set.
+    /// </summary>
+    None = 0x0,
+    
+    /// <summary>
+    /// Write to the output port
+    /// </summary>
+    SetOutputPort = 0xD1,
```

Comment:
I think there are commands that are missing for this to work, see how it's called from the programmer side: https://wiki.osdev.org/A20_Line#Enabling
We would miss 0xAD, 0xD0, 0xAE

Reply:
Indeed, and there is also a simialr command in the BIOS to be implemented.

Reply:
BIOS A20 Gate support implemented. Keyboard missing A20 Gate related commands implemented.